### DISCUSSION TITLE
R plot制图中mtext（）文本方向的问题

### DISCUSSION THREAD

**Discussion by wangb08** [ShixiangWang/self-study#49] [Q&A] [answered]
请问：在plot制图中，使用mtext（）标记坐标轴标题时，当坐标轴为位置4，即右侧时，坐标轴文字方向如何调整，默认情况下是从下往上的，和左侧是一样，但看上去很别扭。
mtext("Number of CVD",4,line=3,at=mean(counts*prop))
![image](https://user-images.githubusercontent.com/42599452/78538290-585e8780-7823-11ea-8bce-57b531301ba1.png)

**Comment by ShixiangWang** [comment, marked as answer]
试下 https://stackoverflow.com/questions/42034786/how-to-rotate-180-degrees-an-mtext-in-r

**Comment by ShixiangWang** [comment]
解决了吗？

**Comment by wangb08** [comment]
非常感谢啊，试了下，效果不是很好，1，会影响原始图像大小，2输出图像时位置不好控制。

**Comment by ShixiangWang** [comment]
如果 base 包不行，就试试 ggplot2 吧，grid 包应该也是可以的。

**Comment by wangb08** [comment]
OK，谢谢
